script/openqa-bootstrap-container

```diff
@@ -7,7 +7,7 @@
 CONTAINER_NAME="openqa1"
 CONTANER_PATH="/var/lib/machines/${CONTAINER_NAME}"
 
-DEFAULT_REPO="http://download.opensuse.org/tumbleweed/repo/oss/"
+: ${DEFAULT_REPO:="http://download.opensuse.org/tumbleweed/repo/oss/"}
```

Comment:
I have never seen this syntax. What about

```
DEFAULT_REPO="${DEFAULT_REPO:="http://download.opensuse.org/tumbleweed/repo/oss/"}"
```

?

Reply:
--> https://github.com/os-autoinst/openQA/pull/1967